### ISSUE TITLE
custom layout is not working in production environment

### ISSUE THREAD

**Issue by junaidcs** [balderdashy/sails#2835]
Hi, 
I'm using sails 0.11.0 and in controller custom layout in not working when i start server with --prod flag(its working without prod flag). e.g
Controller:
```javascript
index: function (req, res) {
  return res.view('administrators/index', {
    layout: 'user'
  });
},
```

I've also tried like:
```javascript
index: function (req, res) {
  res.locals.layout = "user";
  res.view("administrators/index");
},
```
Anyone knows how to solve this issue?

**Comment by tjwebb** [comment]
> not working 

You need to provide more info about what "not working" means.

**Comment by junaidcs** [comment]
I just started the server in production mode with verbose flag and i saw in console that it is including the default layout, so i need to put the path to that custom layout in production like
```javascript
index: function (req, res) {
  return res.view('administrators/index', {
    layout: '../user'
  });
},
```
OR
i have to put user.ejs in administrators directory.
But i'm still confused that why this is working in development mode with layout: 'user' instead of layout: '../user' !

**Comment by CWyrtzen** [comment]
@junaidcs are you still having this problem? Did you find a solution?

**Comment by junaidcs** [comment]
Yes, @CWyrtzen its the same to me when i lift server with `sails lift --prod`.

**Comment by sailsbot** [comment]
Thanks for posting, @junaidcs.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
